v22.2/online-schema-changes.md

```diff
@@ -85,6 +85,93 @@ Declarative schema changer statements and legacy schema changer statements opera
 
 ## Best practices for online schema changes
 
+### Estimate your storage capacity before performing online schema changes
+
+Some schema change operations like adding or dropping columns, or altering primary keys will result in temporary increases to a cluster's storage consumption. These operations will result in consuming up to three times the storage space of the range size while the schema change is being applied, which may cause the cluster to run out of storage space and pause or fail to apply the schema change.
```

Comment:
```suggestion
Some schema change operations, like adding or dropping columns or altering primary keys, will temporarily increase a cluster's storage consumption. Specifically, these operations may temporarily require up to three times more storage space  for the range size while the schema change is being applied, and this may cause the cluster to run out of storage space or fail to apply the schema change.
```